build-config/make/lzo.make

```diff
@@ -9,7 +10,7 @@
 # This is a makefile fragment that defines the build of lzo
 #
 
-LZO_VERSION		= 2.06
+LZO_VERSION		= 2.09
```

Comment:
I uploaded lzo 2.09 to the ONIE mirror:
```
monster-15:/tmp/goo$ wget http://mirror.opencompute.org/onie/lzo-2.09.tar.gz
--2016-08-01 09:03:45--  http://mirror.opencompute.org/onie/lzo-2.09.tar.gz
Resolving mirror.opencompute.org (mirror.opencompute.org)... 23.253.235.92
Connecting to mirror.opencompute.org (mirror.opencompute.org)|23.253.235.92|:80... connected.
HTTP request sent, awaiting response... 200 OK
Length: 594855 (581K) [application/x-gzip]
Saving to: `lzo-2.09.tar.gz'

100%[====================================================================================>] 594,855     1.96M/s   in 0.3s    

2016-08-01 09:03:46 (1.96 MB/s) - `lzo-2.09.tar.gz' saved [594855/594855]
```

and the sha1 is:
```
monster-15:/tmp/goo$ sha1sum lzo-2.09.tar.gz 
e2a60aca818836181e7e6f8c4f2c323aca6ac057  lzo-2.09.tar.gz
```

Let me know if you agree with this version and sha1.